File: PTA/Advanced_Level/1035.cpp
#include<bits/stdc++.h>
using namespace std;
vector<string> v1, v2;
string transform(string s) {
    for(int i = 0; i < s.size(); ++i) {
        if(s[i] == '1') s[i] = '@';
        else if(s[i] == '0') s[i] = '%';
        else if(s[i] == 'l') s[i] = 'L';
        else if(s[i] == 'O') s[i] = 'o';
    }
    return s;
}
int main() {
    int t;  
        cin >> t;
    for(int i = 0; i < t; ++i) {
        string s1, s2;
            cin >> s1 >> s2;
        string s = transform(s2);
        if(s != s2) {
            v1.push_back(s1);
            v2.push_back(s);
        }
    } 
    if(t == 1 && v1.empty()) puts("There is 1 account and no account is modified");
    else if(v1.empty())  printf("There are %d accounts and no account is modified", t);
    else {
        cout << v1.size() << endl;
        for(int i = 0; i < v1.size(); ++i)
            cout << v1[i] << ' ' << v2[i] << endl;
    }
    return 0;
}

File: PTA/Advanced_Level/1035.rs
use std::io::stdin;
fn transform(s: String) -> String {
    let mut res: String = String::new();
    for c in s.chars() {
        if c == '1' {
            res = res + "@";
        } else if c == '0' {
            res = res + "%";
        } else if c == 'l' {
            res = res + "L";
        } else if c == 'O' {
            res = res + "o";
        } else {
            res = res + String::from(c).as_str();        
        }
    }
    res 
}
fn main() {
    let mut input: String = String::new();
    stdin().read_line(&mut input).unwrap();
    let t: u32 = input.trim().parse::<u32>().unwrap();

    let mut v1: Vec<String> = Vec::new();
    let mut v2: Vec<String> = Vec::new();

    input.clear();
    for _ in 0..t {
        stdin().read_line(&mut input).unwrap();
        let temp: Vec<String> = input.trim().split(' ').map(|s| s.to_string()).collect();
        let s = transform(temp[1].clone());
        //println!("{}", s);
        if s != temp[1] {
            v1.push(temp[0].clone());
            v2.push(s);
        }
        input.clear();
        //println!("t {:?} {} {} {}", temp, temp[0], tmep[1], s);       
    }
    if t == 1 && v1.is_empty() {
        println!("There is 1 account and no account is modified");
    } else if v1.is_empty() {
        println!("There are {} accounts and no account is modified", t);
    } else {
        println!("{}", v1.len());
        for i in 0..v1.len() {
            println!("{} {}", v1[i], v2[i]);
        }
    }
    
}